src/Stream.sol

```diff
@@ -228,110 +238,93 @@ contract Stream is IStream, Clone {
     }
 
     /**
-     * @notice Cancel the stream and send payer and recipient their fair share of the funds.
-     * If the stream is sufficiently funded to pay recipient, execution will always succeed.
-     * Payer receives the stream's token balance after paying recipient, which is fair if payer
-     * hadn't fully funded the stream.
+     * @notice Withdraw tokens to recipient's account after the stream has been cancelled.
+     * Execution fails if the requested amount is greater than recipient's withdrawable balance.
+     * Only this stream's payer or recipient can call this function.
+     * @param amount the amount of tokens to withdraw.
+     */
+    function withdrawAfterCancel(uint256 amount) external onlyPayerOrRecipient {
```

Comment:
nit: place this function after the `cancel` function so it's easier to read the file

Reply:
fixed